code.json

```diff
@@ -222,6 +222,53 @@
       "email": "jlaura@usgs.gov"
     },
 
+    "date": {
+      "metadataLastUpdated": "2023-02-23"
+    }
+  }, 
+  {
+    "name": "ale",
+    "organization": "U.S. Geological Survey",
+    "description": "GitLab repository for the Abstraction Layer for Ephemerides package",
+    "version": "1.0.0",
+    "status": "Production",
+
+    "permissions": {
+      "usageType": "openSource",
+      "licenses": [
+        {
+          "name": "Public Domain, CC0-1.0",
+          "URL": "https://code.usgs.gov/astrogeology/ale/-/raw/1.0.0/LICENSE.md"
+        }
+      ]
+    },
+
+    "homepageURL": "https://code.usgs.gov/astrogeology/ale/-/tree/1.0.0",
+    "downloadURL": "https://code.usgs.gov/astrogeology/ale/-/archive/0.8.7/ale-1.0.0.zip",
+    "disclaimerURL": "https://code.usgs.gov/astrogeology/ale/-/raw/1.0.0/DISCLAIMER.md",
+    "repositoryURL": "https://code.usgs.gov/astrogeology/ale.git",
+    "vcs": "git",
+
+    "laborHours": 480,
+
+    "tags": [
+      "Planetary",
+      "Remote Sensing",
+      "Data Processing",
+      "Ephemerides",
+      "Community Sensor Model"
+    ],
+
+    "languages": [
+      "C++",
+      "Python"
+    ],
+
+    "contact": {
+      "name": "Jason Laura",
+      "email": "jlaura@usgs.gov"
```

Comment:
I don't think Jay should remain the point of contact here, either myself or potentially @Kelvinrr